plugins/woocommerce/includes/emails/class-wc-email-admin-payment-gateway-enabled.php

```
@@ -0,0 +1,384 @@
+		 * Output dynamic block content for this email.
+		 *
+		 * Hooked into `woocommerce_email_general_block_content` to render the gateway
+		 * title, security notice, and gateway settings URL inside the ##WOO_CONTENT## area.
+		 *
+		 * @since 10.6.0
+		 * @param bool     $sent_to_admin Whether the email is being sent to admin.
+		 * @param bool     $plain_text    Whether the email is being sent as plain text.
+		 * @param WC_Email $email         The email object.
+		 * @return void
+		 */
+		public function block_content( $sent_to_admin, $plain_text, $email ): void {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$sent_to_admin'. (undefined)

(UnusedFormalParameter)